linux_os/guide/system/software/updating/ensure_redhat_gpgkey_installed/oval/shared.xml

```diff
@@ -24,16 +26,24 @@
           test_ref="test_package_gpgkey-{{{ aux_pkg_version }}}-{{{ aux_pkg_release }}}_installed" />
         </criteria>
       </criteria>
-      <criteria comment="CentOS Vendor Keys" operator="OR">
+      {{%- if product == "rhel6" %}}
+      <criteria comment="CentOS Vendor Keys" operator="AND">
         <criteria comment="CentOS Installed" operator="OR">
```

Comment:
This ` OR criteria` "CentOS Installed" is not needed.

Reply:
Removed